draft-ietf-capport-architecture.xml

```diff
@@ -695,7 +695,29 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
                 link. In such cases, identification could be performed by subnet, such as the /64
                 to which the IP belongs.
             </t>
-           </section>
+         </section>
+         <section anchor="context_free_url" title="Context-free URL">
+           <t>
+             The URLs provided in the API SHOULD contain all the information necessary to render
+             the resources requested: they should be context free so it does not depend on ambient
+             information, such as remote address on the connection. This is to ensure that User
+             Equipment can access URLs from a network other than the one that contains the captive
+             portal. One possible approach is to include the identification of the User Equipment
+             in the URLs.
+           </t>
+           <t>
+             The URLs provided in the API are recommended to be resolved using public DNS, so they
```

Comment:
Should this be normative? (I.e. "RECOMMENDED")?

Reply:
This doc is currently targeting Informational.  We could all-caps recommend here, but it might be good to replicate the message over in the standards track API doc as well.